src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -131,6 +136,7 @@
         }
 
         .datagrid-column {
+            position: relative;
```

Comment:
@jeeyun removed this on purpose, adding it will break the filters again. Please check up with her.